Fabric.Identity.API/scripts/queue-build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+definitionId=$1
+queueBuildResponse=$(curl -X POST -H "Content-Type: application/json" -H "Authorization: Bearer $SYSTEM_ACCESSTOKEN" -d "{ \"definition\": { \"id\": $definitionId } }" https://healthcatalyst.visualstudio.com/DefaultCollection/Fabric/_apis/build/builds?api-version=2.0)
+echo $queueBuildResponse
+
+buildId=$(echo ${queueBuildResponse} | jq .id)
+echo "BuildId: $buildId"
+
+status="\"notStarted\""
+completedStatus="\"completed\""
+while [ "$status" != "$completedStatus" ]; do
```

Comment:
Are we always guaranteed a "completed" status eventually? Or do we want to put a limit on the number / time elapsed in this loop and fail the parent build once that limit is reached?

Reply:
We'll have a timeout setting on the build task to terminate this step and fail the build if we for some reason don't get a "completed" status.